### PR TITLE
Fix Status Bar color, and some minor adjustments in the create channel screen

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
The StatusBar was getting all white when moving to certain screens, in particular Create or Edit Channel.

While fixing this, I realized some other minor style issues, among others, the Autocomplete positioning. I tried to fix those too.

#### Ticket Link
None

#### Release Note
```release-note
Fix Status Bar going white on certain screens like Create or Edit Channel.
Improve Create or Edit channel styles.
Improve Create or Edit channel autocomplete positioning.
```

### CODE REVIEW

**Comment by larkox** [comment]
Missing iOS testing and tweaking.

**Review by enahum — APPROVED** [review]

**Comment by enahum** [comment]
Testing on iOS is crucial before merging

**Comment by larkox** [comment]
> Testing on iOS is crucial before merging

Therefore the Draft status :P

**Comment by larkox** [comment]
The autocomplete is not perfect by all means, but seems somehow better. We can keep iterating on that on other PRs.

**Comment by larkox** [comment]
Tested on the three, yes, and seemed "good enough".

File: app/screens/create_or_edit_channel/channel_info_form.tsx [hunk 1]
```diff
@@ -161,21 +156,25 @@ export default function ChannelInfoForm({
...
-        if (Platform.OS === 'android') {
-            setKeyboardHeight(frames.endCoordinates.height);
-        }
-    }, [scrollHeaderToTop]);
+        setKeyboardPosition(frames.endCoordinates.screenY);
+        setKeyboardHeight(frames.endCoordinates.height);
+    }, []);
 
-    const onKeyboardDidHide = useCallback(() => {
+    const onKeyboardDidHide = useCallback((frames: any) => {
```
Comment (enahum) [inline app/screens/create_or_edit_channel/channel_info_form.tsx:175]: Why are we using any for the frames type?

File: app/screens/navigation.ts [hunk 2]
```diff
@@ -345,7 +345,6 @@ export function goToScreen(name: string, title: string, passProps = {}, options
             right: {enabled: false},
         },
         statusBar: {
-            backgroundColor: null,
```
Comment (enahum) [inline app/screens/navigation.ts:348]: Why this change? I went based on https://wix.github.io/react-native-navigation/next/docs/style-statusBar/#how-to-keep-current-statusbar-color-when-displaying-screens

So if you explain this to me I'd be a happy man

Comment (larkox) [reply app/screens/navigation.ts:348]: Maybe something is not working as expected, but that was the culprit of the white status bar.

I did not think so much through it: it seemed out of place and removing it solved the problem.

Maybe there is no "current status bar color", and somehow the logic fails and set it to white, but that would need deeper look into the react-native-navigation code.

File: app/screens/create_or_edit_channel/channel_info_form.tsx (hunk 1 above)
Comment (larkox) [reply app/screens/create_or_edit_channel/channel_info_form.tsx:175]: KeyboardAwareScrollView expects an Object, so I did not want to make too many assumptions. But yes, I can put the `KeyboardEvent` type.
